src/index.html

```diff
@@ -6,7 +6,37 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="calendar calendar--start-day--mon calendar--month-length--31">
+    <div class="calendar__day calendar__day--1"></div>
+    <div class="calendar__day calendar__day--2"></div>
+    <div class="calendar__day calendar__day--3"></div>
```

Comment:
no needs to add modifier to your calendar__day
```suggestion
    <div class="calendar__day"></div>
    <div class="calendar__day"></div>
    <div class="calendar__day"></div>
```